tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -126,3 +138,23 @@ Verify Cuda Builds Are Completed
         END
         Should Be Equal As Strings    ${pre}[3]    Complete
     END
+
+Verify Link And Run Commands On Prometheus Pod For Jupyterhub And Rhods-Dashboard URLs
+    [Documentation]    Verifies the links and Run curl command on prometheus pod
+    [Arguments]    ${links}
+    ${jupyterhub} =    Set Variable    jupyterhub
+    ${rhods_dashboard} =    Set Variable    rhods-dashboard
+    ${pod_name} =    Find First Pod By Name    namespace=redhat-ods-monitoring    pod_start_with=prometheus-
+    FOR    ${link}    IN    @{links}
+        IF    "${jupyterhub}" in "${link}"
+            ${command} =    Set Variable    curl --insecure ${link}
+            ${output} =    Run Command In Container    namespace=redhat-ods-monitoring    pod_name=${pod_name}
+            ...    command=${command}    container_name=prometheus
+            Should Contain    ${output}    Log in with OpenShift
```

Comment:
you can add msg=Log in with OpenShift should be required to access blackbox-exporter